stroeer/page/section/v1/section_page_service.proto

```diff
@@ -71,12 +71,25 @@ service SectionPageService {
 // Correct paths have a leading and a trailing slash, like `/nachrichten/unterhaltung/`
 // The homepage has the path `/`.
 message GetSectionPageRequest {
+  // valid section_path, with leading and trailing slash
   string section_path = 1;
+
+  // use to page through sections. If unspecified, it will default to `1`.
+  // Paging is 1-based (1 is the first page, there is no page `0`)
+  //
+  // Due to underlying mechanisms and seo requirements, page-size is fixed at 30
+  // The service may return fewer than this value.
+  int32 page = 2;
 }
 
 // Response message when requesting data for a section page.
 // Responds with `NOT_FOUND` if an unknown path is requested, or the path is incorrect.
 message GetSectionPageResponse {
   stroeer.page.section.v1.SectionPage section_page = 1;
-}
 
+  // if request contained a `page`, this page is confirmed here with the same value
+  int32 page = 2;
```

Comment:
what's the benefit of mirroring this value in the response?

Reply:
I thought of two reasons:

1. its kinda render-ready, so you just need to pass the response to your templates.
2. if the user pages beyond the dataset, we can deliver something instead of 404ing? (e.g. pass him the last page if he requests a page that does not exist?)

This is what currently happens on our website `page_9999999` https://www.t-online.de/gesundheit/ernaehrung/id_12289886/page_9999999/ernaehrung-gesundes-essen-ernaehrungsberatung.html

Reply:
seo-jörg says this is wrong: if the users request exceeds the total_pages, we deliver a 404.

Reply:
gonna remove this